nilearn/_utils/tests/test_helpers.py

```diff
@@ -158,3 +159,7 @@ def test_is_plotly_installed():
 
 def test_is_kaleido_installed():
     helpers.is_kaleido_installed()
+
+
+def test_stringify_path():
+    assert isinstance(helpers.stringify_path(Path("foo") / "bar"), str)
```

Comment:
```suggestion
    assert isinstance(helpers.stringify_path(Path("foo") / "bar"), str)
    assert helpers.stringify_path([]) == []
```

Reply:
Shall we add a test with another object to make sure that it does not change the object's type and returns it as it is?

Actually the docstring can also be updated and made more clear for `stringify_path` as it does not always return `str`.

Reply:
I am not sure if this function is really necessary.